src/test/java/org/broadinstitute/consent/http/service/UserServiceTest.java

```diff
@@ -95,15 +96,15 @@ class UserServiceTest {
   private EmailService emailService;
 
   @Mock
-  private DraftService draftService;
+  private DraftServiceDAO draftServiceDAO;
 
 
   private UserService service;
 
   private void initService() {
     service = new UserService(userDAO, userPropertyDAO, userRoleDAO, voteDAO, institutionDAO,
         libraryCardDAO, acknowledgementDAO, fileStorageObjectDAO, samDAO, userServiceDAO, daaDAO,
-        emailService, draftService);
+        emailService, draftServiceDAO);
```

Comment:
If you wanted to get coverage on the new code in `UserService`, you could add a line to `testDeleteUser()` below like
```
    verify(draftServiceDAO).deleteDraftsByUser(u);
```